assets/app/styles/_core.less

```diff
@@ -193,6 +193,10 @@ html, body {
         }
       }
     }
+    .route {
```

Comment:
@liggitt We can, it just seemed weird to me to have an `<h2>` immediately followed by another `<h2>` in the markup.